privdetect/privilegedetect_windows.go

```diff
@@ -41,8 +44,6 @@ const (
 )
 
 var (
-	modAdvapi32 = syscall.NewLazyDLL("Advapi32.dll")
-	procGetTokenInformation = modAdvapi32.NewProc("GetTokenInformation")
 	errERROR_IO_PENDING error = syscall.Errno(errnoERROR_IO_PENDING)
 )
 
```

Comment:
The errnoErr() function is not being used anymore with this change. Please remove it.

Reply:
removed along with the unused variable